test/integration/operator/gardenlet/gardenlet_test.go

```diff
@@ -185,6 +185,10 @@ var _ = Describe("Gardenlet controller test", func() {
 
 			verifyGardenletDeployment(true)
 
+			Eventually(func(g Gomega) {
+				g.Expect(mgrClient.Get(ctx, client.ObjectKeyFromObject(seed), seed)).To(Succeed())
+			}).Should(Succeed())
```

Comment:
I think we could even write:

```suggestion
			Eventually(func() {
				return mgrClient.Get(ctx, client.ObjectKeyFromObject(seed), seed)
			}).Should(Succeed())
```

Reply:
Yup, looks cleaner that way :)